command/ca/init.go

```diff
@@ -411,6 +437,13 @@ func initAction(ctx *cli.Context) (err error) {
 			ui.Println()
 			return nil
 		}
+		// When initializing a linked CA, providing the --acme flag doesn't currently
+		// result in the default ACME provisioner being added. We may want to support this
+		// for ease of use, but this seems to require a bit of refactoring when generating
+		// the full CA configuration with DB initialization.
+		if deploymentType != pki.StandaloneDeployment && addDefaultACMEProvisioner {
+			return fmt.Errorf("adding a default ACME provisioner by providing the --acme flag is not supported with deployment type %q. Please use `step ca provisioner add acme --type ACME` after initializing your CA", deploymentType.String())
```

Comment:
A `\n` would be nice here:
```suggestion
			return fmt.Errorf("adding a default ACME provisioner by providing the --acme flag is not supported with deployment type %q.\nPlease use `step ca provisioner add acme --type ACME` after initializing your CA", deploymentType.String())
```